src/docfx/config/ops/OpsConfig.cs

```diff
@@ -12,8 +12,6 @@ internal class OpsConfig
 {
     public OpsDocsetConfig[] DocsetsToPublish { get; init; } = Array.Empty<OpsDocsetConfig>();
 
-    public HashSet<PathString> RedirectionFiles { get; init; } = new HashSet<PathString>();
```

Comment:
Should we remind user to remove their RedirectionFiles  config? 
e.g. give some warning/suggestions

Reply:
As discussed with @herohua before, it's acceptable not reporting a warning during the build. We will as PM to update the tutorial to inform writers that this config is not needed anymore. But we won't force them to remove this config